internal/io/sql/reader.go

```diff
@@ -61,7 +64,7 @@ func (c *Column) Scan(t interface{}) error {
 	case []uint8:
 		c.data.Strings = append(c.data.Strings, string(v))
 	}
-	return nil
+	return errors.New("ReadSQL Scan", fmt.Sprintf("unsupported column type %s", reflect.TypeOf(t).Name()))
```

Comment:
Picky: No need to call `fmt.Sprintf` here, `errors.New` does the formating for you by doing just that.
This is a general comment.